pandas/tests/internals/test_managers.py

```diff
@@ -68,3 +84,19 @@ def test_series_creation():
     result = s_array._as_manager("block")
     assert isinstance(result._mgr, SingleBlockManager)
     tm.assert_series_equal(result, s_array)
+
+
+@pytest.mark.parametrize("manager", ["block", "array"])
+def test_array_manager_depr_env_var(manager):
```

Comment:
Can you mark this as `single_cpu`?

Reply:
Happy to - but I'm curious as to why.

Reply:
Just precautionary, but just to ensure xdist workers don't use even more resources by spawning subprocesses